### PR TITLE
executors: runlock panels with milliseconds units

### PR DESCRIPTION
**Pull request by Strum355**
Sets the units for the executor runlock panels to milliseconds instead of none

## Test plan

Ran locally, all OK

### CODE REVIEW

File: doc/admin/observability/dashboards.md [hunk 1]
```diff
@@ -17400,7 +17400,7 @@ Query: `sum(increase(src_executor_processor_errors_total{queue=~"${queue:regex}"
 
 ### Executor: Run lock contention
 
-#### executor: executor_run_lock_wait_total
+#### executor: (executor|sourcegraph-code-intel-indexers|executor-batches|sourcegraph-executors)
```
Comment (eseliger) [inline doc/admin/observability/dashboards.md:17403]: this looks weird?

File: dev/codeintel-qa/cmd/upload/state.go [hunk 2]
```diff
@@ -119,6 +119,12 @@ func monitor(ctx context.Context, repoNames []string, uploads []uploadMeta) erro
 					} else {
 						fmt.Printf("DUMP:\n\n%s\n\n\n", out)
 					}
+					out, err = exec.Command("docker", "exec", containerName, "sh", "-c", "pg_dump -U postgres -d sourcegraph -a --column-inserts --table='lsif_configuration_policies'").CombinedOutput()
```
Comment (eseliger) [inline dev/codeintel-qa/cmd/upload/state.go:122]: is this related?

Comment (efritz) [reply dev/codeintel-qa/cmd/upload/state.go:122]: DUMP

Comment (Strum355) [reply dev/codeintel-qa/cmd/upload/state.go:122]: whoops lol

**Review by eseliger — APPROVED** [review]
